tests/core/test_core.py

```diff
@@ -70,6 +74,27 @@ def test_run_core_as_a_service_standalone_mode(self):
         assert isinstance(_OrchestratorFactory._dispatcher, _StandaloneJobDispatcher)
         assert core._dispatcher.is_running()
         assert _OrchestratorFactory._dispatcher.is_running()
+        core.stop()
+
+    def test_core_service_can_only_be_run_once(self):
+        core_instance_1 = Core()
+        core_instance_2 = Core()
+        Config.configure_job_executions(mode=JobConfig._DEVELOPMENT_MODE)
```

Comment:
This line should not be necessary. THe contest should have cleaned the Config. Am I right?

Reply:
Yes, we can remove those